docs/source/spawner-types.md

```diff
@@ -147,3 +147,19 @@ c.DockerSpawner.host_ip = "0.0.0.0"
 This will configure DockerSpawner and SystemUserSpawner to get
 the container IP address and port number using the `docker port`
 command.
+
+## Using Podman
+
+Podman is an alternative to Docker for running containers, and supports running containers without root access.
+It is not supported by JupyterHub, but it can be used with DockerSpawner by [running a podman service](https://docs.podman.io/en/latest/markdown/podman-system-service.1.html):
```

Comment:
Wasn't sure what to say here- not supported, unofficially supported, partially supported?

Reply:
maybe "not officially supported"? I think "not supported" is accurate, but usually gives the impression of "doesn't work," whereas "not officially supported" is perhaps more likely to give the right impression?

Or if we do want to support it eventually, you could say "support for podman is experimental" to get at the fact that we might not be ready yet, but enthusiasts are welcome to try it out and provide feedback about what works and doesn't.

Reply:
Feel free to pick language that feels right to you and go ahead and merge.

Reply:
Thanks, I went with ""not officially supported"